pkg/collector/adapters/config_validate.go

```diff
@@ -27,7 +27,7 @@ var (
 
 //Following Otel Doc: Configuring a receiver does not enable it. The receivers are enabled via pipelines within the service section.
 //ConfigValidate returns all receivers, setting them as true for enabled and false for non-configured services in pipeline set.
-func ConfigValidate(logger logr.Logger, config map[interface{}]interface{}) (map[interface{}]bool, error) {
+func GetEnabledReceivers(logger logr.Logger, config map[interface{}]interface{}) (map[interface{}]bool, error) {
 	cfgReceivers, ok := config["receivers"]
 	if !ok {
 		return nil, ErrNoReceivers
```

Comment:
I like this, but it should still never return an error, ie, never fail. If the data is bad, then no receivers are enabled, so, just return nil.

Reply:
Thank you @jpkrohling. I've developed to return only the enabled receivers but forget to change the function signature and all the components related.